src/p5.dance.js

```diff
@@ -83,6 +83,9 @@ module.exports = class DanceParty {
 
     this.world.keysPressed = new Set();
 
+    this.world.spriteGroupsCalledToChangeMove = [];
+    this.world.spriteStyles = [];
```

Comment:
I wonder if we should name/comment/group these kinds of variables so that future users of this code realize that they are related to validation, rather than used internally.

Reply:
Good call. Added a comment for the ai validation variables as well.